modules/net-vpc-firewall-yaml/README.md

```diff
@@ -4,47 +4,56 @@ This module allows creation and management of different types of firewall rules
 
 Yaml abstraction for FW rules can simplify users onboarding and also makes rules definition simpler and clearer comparing to HCL.
 
-Nested folder structure for yaml configurations is supported, which allows better and structured code management. 
+Nested folder structure for yaml configurations is supported, which allows better and structured code management for multiple teams and environments. 
 
 ## Example
 
 ### Terraform code
 
 ```hcl
 module "prod-firewall" {
-  source      = "./modules/net-vpc-firewall-yaml"
-  project_id  = "my-prod-project"
-  network     = "my-prod-network"
-  config_path = "./prod"
+  source = "./modules/net-vpc-firewall-yaml"
+  
+  project_id         = "my-prod-project"
+  network            = "my-prod-network"
+  config_directories = [
+    "./prod",
+    "./common"
+  ]
+
   log_config  = {
     metadata = "INCLUDE_ALL_METADATA"
   }
 }
 
 module "dev-firewall" {
-  source      = "./modules/net-vpc-firewall-yaml"
-  project_id  = "my-dev-project"
-  network     = "my-dev-network"
-  config_path = "./dev"
+  source = "./modules/net-vpc-firewall-yaml"
+  
+  project_id         = "my-dev-project"
+  network            = "my-dev-network"
+  config_directories = [
+    "./prod",
```

Comment:
Should this be `./dev`?

Reply:
Fixed the readme, good catch. I will think about hierarchical rules

Reply:
@ludoo I have added a bit more of context around dep/prod/common folders with FW rules, also noted hierarchical FW rules.

Reply:
Noted hierarchical FW rules in the example readme, let me know if we want to duplicate it in the module readme as well.